web/src/enterprise/campaigns/apply/VisibleChangesetSpecNode.story.tsx

```diff
@@ -33,7 +33,19 @@ export const visibleChangesetSpecStories: Record<string, VisibleChangesetSpecFie
             baseRef: 'master',
             headRef: 'cool-branch',
             body: 'Body text',
-            commits: [{ message: 'Commit message' }],
+            commits: [
+                {
+                    subject: 'Commit message',
```

Comment:
It's really easy to confuse this with a header when looking at the story in the Storybook.


```suggestion
                    subject: 'This is the first line of the commit message',
```

I always try to use real looking data in my examples vs. data that describes what the data is.

Good:

```go
func add(a, b int) (sum int) { return a + b }
```

Bad:

```go
func function(param1, param2 type) (returnValue type) { return param1 + param2 }
```

You get the idea :)